packages/cli/src/commands/apps/index.ts

```diff
@@ -0,0 +1,124 @@
+import {ux} from '@oclif/core'
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import * as _ from 'lodash'
+import color from '@heroku-cli/color'
+import {SpaceCompletion} from '@heroku-cli/command/lib/completions'
+import {App} from '../../lib/types/app'
+
+function annotateAppName(app: App) {
+  let name = `${app.name}`
+  if (app.locked && app.internal_routing) {
+    name = `${app.name} [internal/locked]`
+  } else if (app.locked) {
+    name = `${app.name} [locked]`
+  } else if (app.internal_routing) {
+    name = `${app.name} [internal]`
+  }
+
+  return name
+}
+
+function regionizeAppName(app: App) {
+  const name = annotateAppName(app)
+  if (app.region && app.region.name !== 'us') {
+    return `${name} (${color.green(app.region.name)})`
+  }
+
+  return name
+}
+
+function listApps(apps: Heroku.App) {
+  apps.forEach((app: App) => ux.log(regionizeAppName(app)))
+}
+
+function print(apps: Heroku.App, user: Heroku.Account, space: string, team: string) {
+  if (apps.length === 0) {
+    if (space) ux.log(`There are no apps in space ${color.green(space)}.`)
+    else if (team) ux.log(`There are no apps in team ${color.magenta(team)}.`)
+    else ux.log('You have no apps.')
+  } else if (space) {
+    ux.styledHeader(`Apps in space ${color.green(space)}`)
+    listApps(apps)
+  } else if (team) {
+    ux.styledHeader(`Apps in team ${color.magenta(team)}`)
+    listApps(apps)
+  } else {
+    apps = _.partition(apps, (app: App) => app.owner.email === user.email)
+    if (apps[0].length > 0) {
+      ux.styledHeader(`${color.cyan(user.email!)} Apps`)
+      listApps(apps[0])
+    }
+
+    const columns = {
+      name: {get: regionizeAppName},
+      email: {get: ({owner}: any) => owner.email},
+    }
+
+    if (apps[1].length > 0) {
+      ux.styledHeader('Collaborated Apps')
+      ux.table(apps[1], columns, {'no-header': true})
+    }
+  }
+}
+
+export default class AppsIndex extends Command {
+  static description = 'list your apps'
+  static topic = 'apps'
+  static aliases = ['apps', 'list', 'apps:list']
```

Comment:
No need to alias itself
```suggestion
  static aliases = ['list', 'apps:list']
```